sdc/datatypes/hpat_pandas_stringmethods_functions.py

```diff
@@ -1016,6 +1016,12 @@ def hpat_pandas_stringmethods_strip(self, to_strip=None):
         Series elements are expected to be Unicode strings. Elements cannot be `NaNs`.
 """
 
+limitation_none_unsupported = """
+        Limitations
+        -----------
+        Methods converts `None` in `NaN`.
+"""
```

Comment:
I would write something like that (+ rename the variable):
```suggestion
limitation_nans_supported = """
        Limitations
        -----------
        All values in Series equal to `None` are converted to `NaNs`.
"""
```